main.nf

```diff
@@ -854,13 +909,13 @@ if (params.protocol != 'amplicon') {
     process IVAR_TRIM {
         tag "$sample"
         label 'process_medium'
-        publishDir "${params.outdir}/variants/ivar", mode: params.publish_dir_mode,
+        publishDir "${params.outdir}/variants/bowtie2", mode: params.publish_dir_mode,
```

Comment:
Where does `bowtie2` come in here and below?

Reply:
I changed this because it just means that all of the BAMs generated initially via Bowtie2 and thereafter via post-processing steps (`IVAR_TRIM` and `PICARD_MARKDUPLICATES`) are always in the same folder. e.g. for metagenomics data they will be in `bowtie2` and for amplicon data they will be in `ivar` which is a bit confusing.